bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_utils.py

```diff
@@ -35,6 +44,25 @@ def _kwargs_to_roocmdargs(*args, **kwargs):
     return args, {}
 
 
+def _string_to_root_attribute(value, lookup_map):
```

Comment:
Perhaps some docs here would be good? What kind of lookups in the ROOT module does this function handle and for what purpose?

Reply:
Hi @Harshalzzzzzzz, I think this is not resolved yet. You added this explanation:
```Python
"""Helper function to pythonize arguments based on the matplotlib color/style conventions."""
```
I think it's better to have a more general explanation with an example, and a dedicated sentence to explain what the lookup map is for (having a map to translate matplotlib color/linestyle names to ROOT color/linestyle names is indeed one application). The example could be that you want to make it possible to pass string values instead of enum values that have to be retrieved from the ROOT module:
```Python
pdf.plotOn(frame, LineColor=ROOT.kOrange)
pdf.plotOn(frame, LineColor="kOrange")
```

Reply:
Yes sir, in the previous comment I have mentioned a query, should this be in a docstring?